docs/current_docs/manuals/developer/debugging.mdx

```diff
@@ -63,13 +77,4 @@ The above gives a lot of useful information:
 - The user inputs after being converted to more complex types (structuring)
 - The function's result before and after serialization
 
-Additionally, with `--debug` the GraphQL query that the CLI produced is also shown.
```

Comment:
I also don't see the GraphQL query any longer in the output, so I'm unsure if this statement is still valid. WDYT @levlaz @helderco ?

Reply:
The query still shows, but it's no longer at the end, you'll have to search for it.